docs-ui/stories/components/form-fields.stories.js

```diff
@@ -283,100 +281,6 @@ SelectFieldInFieldLabel.parameters = {
   },
 };
 
-export const CompactSelectField = props => (
-  <CompactSelect
-    defaultValue="opt_one"
-    options={[
-      {value: 'opt_one', label: 'Option One'},
-      {value: 'opt_two', label: 'Option Two'},
-    ]}
-    {...props}
-  />
-);
```

Comment:
Moving the `CompactSelect` and `CompositeSelect` stories out of `Forms / Fields` since they are not really form fields — only `SelectControl` is.